pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/StandardTextValues.groovy

```diff
@@ -20,7 +20,7 @@ package com.linkedin.gradle.python.util
  * are being moved to this enum.  Its an Enum rather than an interface because according to CodeNarc, interfaces of nothing
  * but constants is now taboo.
  */
-enum StandardTextValues {
+public enum StandardTextValues {
```

Comment:
This is a Groovy file. It's public by default and IDE will warn about unnecessary qualifier. If you want to make it Java, though, this would be needed.